src/test/java/uk/gov/hmcts/reform/wataskconfigurationtemplate/dmn/CamundaTaskWaPermissionTest.java

```diff
@@ -72,11 +90,42 @@ void given_multiple_event_ids_should_evaluate_dmn(String caseData,
         MatcherAssert.assertThat(dmnDecisionTableResult.getResultList(), is(expectation));
     }
 
+    @SuppressWarnings("checkstyle:indentation")
+    @Test
+    void given_reviewHearingBundle_taskType_when_evaluate_dmn_then_it_returns_expected_rule() {
```

Comment:
this test is failing as the taskType reviewHearingBundle only returns one row. See DMN.